package.json

```diff
@@ -130,6 +130,6 @@
     "initVersion": "7.25.1"
   },
   "trigger.dev": {
-    "endpointId": "captable-A0-I"
+    "endpointId": "opencap-WhPZ"
```

Comment:
Do we need to hardcode this? Each contributors and env will have different ids

Reply:
No, i will fix it.